schema.sql

```diff
@@ -7,12 +7,12 @@
 #
```

Comment:
Did this commit get added to the MR by accident, it's not been merged into master yet?

Reply:
No. It's here because I've based this new branch on the current `fixDbSchemaCreation` one. Once the other branch gets merged, these two commits will (or at least, should) automatically disappear from this pull request.

Reply:
So, GitHub is less savvy than BitBucket when merging pull requests with identical commits... :-( I'll remember that for next time.

Reply:
Ha yeah, don't get me started with all the things that are wrong with github 😬